JobConfig/validation/stoppedMuonsSingleStage.fcl

```diff
@@ -128,8 +128,10 @@ physics.producers.g4run.Mu2eG4CommonCut: {
 
 physics.producers.g4run.SDConfig.enableSD: [ virtualdetector ] # activate just the explicitly listed SDs
 
-# Not too coarse - we care about stopped muon positions.
-physics.producers.g4run.physics.minRangeCut : 0.010 # mm
+# Coarser cuts per e-mail discussion with Krzysztof Genser on 2022-02-18
+physics.producers.g4run.physics.minRangeCut : 1. # mm
+physics.producers.g4run.physics.protonProductionCut : 1. # mm
+physics.producers.g4run.physics.turnOnRadioactiveDecay : true
```

Comment:
This should be turnOffRadioactiveDecay, not On, 
as the idea is that those muons do not come from the radioactive decays.
One should probably check if one would see a statistically significant difference in the number of stopped muons, while seeing CPU savings.